package.json

```
@@ -21,6 +21,7 @@
     "lint": "eslint src/*.js test/*.js",
     "test-generate-mo": "msgfmt test/fixtures/latin13.po -o test/fixtures/latin13.mo & msgfmt test/fixtures/utf8.po -o test/fixtures/utf8.mo & msgfmt test/fixtures/obsolete.po -o test/fixtures/obsolete.mo",
     "test": "mocha",
+    "test:coverage": "npx c8 --check-coverage npm run test",
     "preversion": "npm run lint && npm test",
     "postversion": "git push && git push --tags",
     "prepublish": "tsc && npm run lint && npm run test"
```

```suggestion
    "prepublishOnly": "tsc && npm run lint && npm run test"
```

in that way the hook will be fired only before publish but not during clone